src/components/body/BodyMainPanel.tsx

```diff
@@ -53,12 +60,27 @@ const BodyMainPanel: React.FC<IProps> = ({
     [styleTop, styleLeft, dataLength, bodyRowHeight, bodyContentWidth]
   );
 
+  const onScroll: React.EventHandler<any> = throttle(() => {
+    const scrollTop = scrollContentRef.current?.scrollTop || 0;
+    const scrollLeft = scrollContentRef.current?.scrollLeft || 0;
+    layoutDispatch({
+      type: LayoutContextActionTypes.SET_SCROLL,
+      scrollTop,
+      scrollLeft,
+    });
+  });
```

Comment:
Please use useCallback or useMemo for memoization.
Otherwise, the onScroll function will be recreated on every render.

Ref: https://reactjs.org/docs/hooks-reference.html#usecallback